resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -89,17 +84,14 @@
 		"id": "neutral_interaction4",
 		"interactions": [
 			"m_c stands tall when r_c walks by.",
-			"m_c doesn't mind r_c staring.",

```

Comment:
Removed

Reply:
Thought this one sounded a bit creepy, so I removed it. LOL